eng/sdk_files/Emscripten.Cache.targets

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <Target Name="_EmscriptenCacheSdkOverrideWasmCachePath" BeforeTargets="_PrepareForWasmBuildNative" Condition="'$(WasmBuildNative)' == 'true'">
+    <PropertyGroup>
+      <WasmCachePath Condition="'$(WasmCachePath)' == ''">$([MSBuild]::NormalizeDirectory($(EmscriptenCacheSdkCacheDir)))</WasmCachePath>
```

Comment:
@radical do I need quotes here? how will WasmApp.Native.targets deal with spaces?

Reply:
looks like we don't have quotes for `EmscriptenPythonToolsPath` and that seems to work alright

Reply:
Quote the path where we used the property, instead of here.